site/en/docs/extensions/reference/sidePanel/index.md

```diff
@@ -182,34 +213,38 @@ Navigating to the side panel menu
   </figcaption>
 </figure>
 
-Using the action toolbar icon
-: Users can open the side panel by clicking on [the action icon](#open-action-icon) if it's enabled.
-
-Using a keyboard shortcut
```

Comment:
Did you intentionally drop this paragraph?

Reply:
Yeah I felt like it fell under "user gesture". I'll explicitly list it under **Open through a user gesture**